nsxt/resource_nsxt_edge_transport_node.go

```diff
@@ -1016,7 +1136,7 @@ func getIPAssignmentFromSchema(ipAssignmentList interface{}) (*data.StructValue,
 	converter := bindings.NewTypeConverter()
 
 	for _, ia := range ipAssignmentList.([]interface{}) {
-		iaType, iaData, err := getIPAssignmentData(ia.(map[string]interface{}))
+		iaType, iaData, err := getIPAssignmentData(ia.(map[string]interface{}), ipAssignmentTypes)
```

Comment:
I realized that IPAssignmentSpec also supports StaticIpMacListSpec, now we're adding it to the v6 assignementSpec, but the v4 one does not have it. Probably we should add it.

Reply:
Good point, added `static_ip_mac` option to ipv4, and also the missing `no_ipv4` option